src/instructlab/config/init.py

```diff
@@ -112,12 +112,15 @@ def init(
         config_file,
     )
     if overwrite_profile:
-        click.echo(
-            f"Generating `{DEFAULTS.CONFIG_FILE}` and `{DEFAULTS.TRAIN_PROFILE_DIR}`..."
-        )
+        click.echo()
```

Comment:
nit: Include as a \n on the next echo

Same for the other echo()s below

Reply:
will try later